onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -200,104 +206,131 @@ Status TreeEnsembleCommon<InputType, ThresholdType, OutputType>::Init(
 
   // filling nodes
 
+  std::vector<TreeNodeElementId> node_tree_ids(nodes_treeids.size());
+  std::vector<int64_t> truenode_ids(nodes_treeids.size());
+  std::vector<int64_t> falsenode_ids(nodes_treeids.size());
```

Comment:
Consider using reserve() instead of preallocating.

Reply:
done